conf/vlan_filters.conf.defaults

```diff
@@ -18,3 +19,69 @@ scopes = RegisteredRole
 action.0=modify_node: mac, $mac, status = unreg, autoreg = no
 role = registration
 
+[EXAMPLE_No_Internet_for_lunch]
+run_actions=enabled
+status=disabled
+description=revent a device from connecting when its role is "default", the SSID is "SECURE", the current time is between 11am and 2pm, from Monday to Friday and as a registered device
+scopes=RegisteredRole
+role=REJECT
+condition=node_info.category == "default" && ssid == "SECURE" && time_period(time, "wd {Mon Tue Wed Thu Fri} hr {11am-2pm}")
+
+[EXAMPLE_Trigger_event_if_user]
+run_actions=enabled
+status=disabled
+description=Trigger a security event if the SSID is OPEN and the owner is igmout
+scopes=RegisteredRole
+action.0=trigger_security_event: mac, $mac, tid, 12345, type, CUSTOM
+condition=owner.pid == "igmout" && ssid == "OPEN"
+
+[EXAMPLE_Autoregister_if_user]
+run_actions=enabled
+status=disabled
+description=autoregister the device and assign the role default to each device where the username is igmout.
```

Comment:
autoregister -> Autoregister
the role default -> the default role
where the username -> when the username